### PR TITLE
pagination status bar component

### PR DESCRIPTION
**Pull request by ValeryiaMIRON**
- add select with page size
- add progressIndicator
Снимок экрана 2021-11-19 в 1 08 25 PM

https://trello.com/c/yeVJNmeC/156-fe-add-ag-grid-status-bar-panels-components-to-pagination

### CODE REVIEW

File: src/pages/candidates/Candidates.jsx [hunk 1]
```diff
@@ -26,15 +29,16 @@ const Candidates = () => {
...
+    useState(true);
   const open = !!anchorEl;
   const { id } = useParams();
 
   const listOfCandidates = useSelector((state) => state.candidates.candidates);
 
   const dispatch = useDispatch();
   const requestBody = {
-    pageSize: 20,
+    pageSize: 50,
```
Comment (itsnotpolza) [inline src/pages/candidates/Candidates.jsx:41]: why is page size static if you can configure it in the table

Comment (ValeryiaMIRON) [reply src/pages/candidates/Candidates.jsx]: ok

File: src/pages/candidates/Candidates.jsx [hunk 2]
```diff
@@ -43,6 +59,13 @@ const Candidates = () => {
     dispatch(fetchCandidateList(requestBody));
   }, []);
 
+  if (!listOfCandidates.length) return <ProgressIndicator />;
```
Comment (itsnotpolza) [inline src/pages/candidates/Candidates.jsx:62]: what if there are no candidates on an internship yet? Will it just keep 'loadng'?

Comment (ValeryiaMIRON) [reply src/pages/candidates/Candidates.jsx]: done

**Review by nsukhotskaya — DISMISSED** [review]

File: src/pages/candidates/Candidates.jsx [hunk 3]
```diff
@@ -26,22 +35,38 @@ const Candidates = () => {
...
+    pageSize: 100000,
     pageNumber: 1,
     internshipId: id,
   };
 
   useEffect(() => {
     dispatch(fetchCandidateList(requestBody));
   }, []);
+  
+  if (!listOfCandidates.length && listOfCandidates.length !== 0 ) {
```
Comment (nsukhotskaya) [inline src/pages/candidates/Candidates.jsx:62]: !listOfCandidates.length && listOfCandidates.length !== 0
Always return false

Comment (ValeryiaMIRON) [reply src/pages/candidates/Candidates.jsx]: yes

**Review by nsukhotskaya — APPROVED** [review]

**Review by svtslv17 — APPROVED** [review]

**Pull request merged**